### ISSUE TITLE
login never ends and do not return any error

### ISSUE THREAD

**Issue by realtebo** [meganz/MEGAcmd#630]
I can login using email and password in my browser

I tried to login from my debian linux 11 server.

I launched mega-cmd
the used 

     login email password

Simply, no output, no results, no error.. and online i can't see the session opened, so login is not working.

Can I diagnose in some way?

**Comment by realtebo** [comment]
from command line if i try

    mega-whoami 

it prints 

    Resuming session...

and also this never ends ...

I tried mega-logout from cmdline. It worked

I tried 

      mega-login user@domain.com 'passwordredacted' -vvvvvvvvvvvv

(with valid pair of  email/password) and again it never ends and do not write nothing at all. Damn

**Comment by realtebo** [comment]
Still not able to login. From web we have no problems. From a different account we have no problem

this is the log

`[API:verbose: 07:45:22]  Megaclient exec is pending resolutions. scpaused=0 stopsc=0 mBlocked=0 jsonsc.pos= syncsup=1 statecurrent=0 syncadding=0 syncactivity=0 syncdownrequired=0 syncdownretry=0
[API:debug: 07:45:25] DNS cache hit for g.api.mega.co.nz (IPv6) 2a0b:e46:1:100::11
[API:debug: 07:45:25] cs POST target URL: https://g.api.mega.co.nz/cs?id=pyaolphnow&ak=BdARkQSQ&v=2
[API:debug: 07:45:25] cs Sending 160: [{"a":"us","user":"......... user redacted ...","uh":"yg0xV9MD7y91988hJKBQcA","sek":"rpkzRJ-TJLoPkKSNNyW5lw","si":"VrW5K3bMiQC8iddgfz61k84ZdIvyYBoEGR7rQZrhM-I"}]
[API:debug: 07:45:25] Using the IP of the hostname: [2a0b:e46:1:100::11]
[API:verbose: 07:45:25]  Megaclient exec is pending resolutions. scpaused=0 stopsc=0 mBlocked=0 jsonsc.pos= syncsup=1 statecurrent=0 syncadding=0 syncactivity=0 syncdownrequired=0 syncdownretry=0
[API:debug: 07:45:25] Informing cURL of timeout reached for 2 at 6797937
[API:verbose: 07:45:25] cs cURL: Found bundle for host 2a0b:e46:1:100::11: 0x7f1bfc0b2a40 [serially]
[API:verbose: 07:45:25] cs cURL: Re-using existing connection! (#1) with host 2a0b:e46:1:100::11
[API:verbose: 07:45:25] cs cURL: Connected to 2a0b:e46:1:100::11 (2a0b:e46:1:100::11) port 443 (#1)
[API:verbose: 07:45:25] cs cURL: We are completely uploaded and fine
[API:debug: 07:45:25] Adding curl socket 76 to 1
[API:verbose: 07:45:25]  Megaclient exec is pending resolutions. scpaused=0 stopsc=0 mBlocked=0 jsonsc.pos= syncsup=1 statecurrent=0 syncadding=0 syncactivity=0 syncdownrequired=0 syncdownretry=0
[API:verbose: 07:45:25] cs cURL: Mark bundle as not supporting multiuse
[API:verbose: 07:45:25] cs Header: HTTP/1.1 200 OK
[API:verbose: 07:45:25] cs Header: Content-Type: application/json
[API:verbose: 07:45:25] cs Header: Access-Control-Allow-Origin: *
[API:verbose: 07:45:25] cs Header: Access-Control-Allow-Headers: Content-Type, MEGA-Chrome-Antileak
[API:verbose: 07:45:25] cs Header: Access-Control-Expose-Headers: Original-Content-Length
[API:verbose: 07:45:25] cs Header: Access-Control-Max-Age: 86400
[API:verbose: 07:45:25] cs Header: Cache-Control: no-store
[API:verbose: 07:45:25] cs Header: Original-Content-Length: 2
[API:verbose: 07:45:25] cs Header: Content-Length: 2
[API:verbose: 07:45:25] cs Header: Connection: keep-alive
[API:verbose: 07:45:25] cs cURL: Connection #1 to host 2a0b:e46:1:100::11 left intact
[API:debug: 07:45:25] Removing socket 76
[API:debug: 07:45:25] CURLMSG_DONE with HTTP status: 200 from g.api.mega.co.nz - [2a0b:e46:1:100::11]
[API:debug: 07:45:25] cs Received 2: -3
[API:warn: 07:45:25] Retrying cs request in 92 ds
`

**Comment by polmr** [comment]
Hi @realtebo , does that issue persist, the log indicates a temporary lock in your account. Usually solved in a timely manner. If so, please, contact support@mega.nz, indicating your actual logging and the info here (have this issue referenced also, please)

**Comment by realtebo** [comment]
Yes... I did .. and support@mega.nz told me to open an issue here.

anyway, problem is 'solved' because _after waiting for over 15 minutes_ the login succeded and so, until next vps reboot, I will' never face again the problem.
